Havit.Blazor.Components.Web.Bootstrap.Documentation/Services/ApiHelper.cs

```diff
@@ -71,6 +71,16 @@ public static Type GetType(string typeName)
 		}
 		catch { }
 
+		try
+		{
+			result = typeof(HxButton).Assembly.GetTypes().Where((t) => t.FullName.Contains(typeName)).FirstOrDefault();
+			if (result is not null)
```

Comment:
Do not use `.Where(predicate).FirstOrDefault()`, there is a `.FirstOrDefault(predicate)` overload to be used.